mxcube3/ui/components/SampleView/MotorInput.js

```diff
@@ -65,6 +66,7 @@ export default class MotorInput extends React.Component {
                 step={step}
                 defaultValue={valueCropped}
                 name={motorName}
+                disabled={this.props.state === 4 || this.props.state === 3 || this.props.state <= 1}
```

Comment:
I didn't experience the same problem as you describe, maybe it has something to do with the browser: I am using Firefox here I guess you're running Chrome ?

It's not very nice that it is possible to type on a "logically" read-only field.

In any case I don't see the fact that focus moves away to be very problematic in this particular situation. I advocate for accepting this PR and solving this problem in another one.